README.md

```diff
@@ -7,10 +7,21 @@ desktop applications on Linux.
 
 See https://flatpak.org/ for more information.
 
+Community discussion happens in [#flatpak on Freenode](irc://freenode.net:6697/flatpak) and on [the mailing list](https://lists.freedesktop.org/mailman/listinfo/flatpak).
```

Comment:
ircs://chat.freenode.net/flatpak

Reply:
Yeah I figured it's not a big deal that the link isn't visible on GitHub because most people will just go find it in their IRC client anyway